### ISSUE TITLE
3 day averages should exclude most recent day

### ISSUE THREAD

**Issue by konvolution** [kennygrant/coronavirus#7]
It looks like the 3 day averages is skewed by including the most recent day, which is often missing data. I really like this visualization otherwise, mainly because it feels so light and fast. Thank you for making it.

**Comment by kennygrant** [comment]
Yes I agree, this isn't looking right. I think I'll move it to a 7 day average as well.
